agent/hibagent

```diff
@@ -497,6 +500,36 @@ def adjust_pm_timeout(timeout):
         log("Failed to adjust pm_freeze_timeout to %d. Error: %s" % (timeout, str(e)))
         exit(1)
 
+def get_imds_token(seconds=21600):
+    """ Get a token to access instance metadata. """
+    log("Requesting new IMDSv2 token.")
+    request_header = {'X-aws-ec2-metadata-token-ttl-seconds': '{}'.format(seconds)}
```

Comment:
Should we specify retry value:
TOKEN=$(curl --silent --fail --retry 3 -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600")

Reply:
Here is a Python code implementation for  retries over request.put() :
https://code.amazon.com/packages/AdaptDHAL/blobs/ea7890aad289411b5badfa77d48a05cbda95eedd/--/src/adapt_dhal/devices/features/packages/dlipowerswitch/dlipowerswitch.py#L66